docs/release-notes/cloud/cloud-2023-04.md

```diff
@@ -20,7 +20,7 @@ These release notes are for the Codacy Cloud updates during April 2023.
 ## Bug fixes
 
 -   It's now possible to see and change the repository settings for repositories without commits or before Codacy finishes performing the first analysis. (TS-320)
--   Now, the re-analysis button is disabled unless the current user is either the committer and a member of the organization, or a repository admin. (PLUTO-49)
+-   The button to reanalyze commits to your branches or pull requests is now disabled if the committer isn't part of the organization. (PLUTO-49)
```

Comment:
The part about the branches is redundant since commits always belong to a branch and pull requests also correspond to a branch. The sentence can be simplified like this:

```suggestion
-   The button to reanalyze commits or pull requests is now disabled if the committer isn't part of the organization. (PLUTO-49)
```